resources/vue/src/components/account/common/AccountMain.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <router-view></router-view>
```

Comment:
This should not be a separate component. You can create pages folder where you can combine all of the pages you are planning to have. This will simplify routing configuration, and will give you and your team clarity of what pages does this application have.